src/pharmpy/tools/ruvsearch/results.py

```diff
@@ -48,29 +49,41 @@ def calculate_results(models):
         model_params = []
         for model in iteration_models:
             name = model.name
-            dofv = base_ofv - model.modelfit_results.ofv
-            if name.startswith('IIV_on_RUV'):
-                param = {'omega': round(model.modelfit_results.parameter_estimates["IIV_RUV1"], 6)}
-            elif name.startswith('power'):
-                param = {'theta': round(model.modelfit_results.parameter_estimates["power1"], 6)}
-            elif name.startswith('time_varying'):
-                param = {
-                    'theta': round(model.modelfit_results.parameter_estimates["time_varying"], 6)
-                }
-            else:
-                param = {
-                    'sigma_add': round(model.modelfit_results.parameter_estimates["sigma_add"], 6),
-                    'sigma_prop': round(
-                        model.modelfit_results.parameter_estimates["sigma_prop"], 6
-                    ),
-                }
-            a = name.split('_')
-            name = '_'.join(a[0:-1])
-
-            model_name.append(name)
-            model_dofv.append(dofv)
-            model_params.append(param)
+            if model.modelfit_results.ofv:
```

Comment:
Perhaps 

`if model.modelfit_results is not None and model.modelfit_results.ofv is not None:`